src/QuirksCheck.cpp

```diff
@@ -0,0 +1,120 @@
+/** 
+ * Copyright (C) 2023 Eric Helgeson
+ * Portions ZuluSCSI™ - Copyright (c) 2023 Rabbit Hole Computing™
+ * 
+ * This file is licensed under the GPL version 3 or any later version. 
+ * It is derived from BlueSCSI_platform_config_hook.cpp in BluSCSI-v2
+ * 
+ * https://www.gnu.org/licenses/gpl-3.0.html
+ * ---- 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version. 
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details. 
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+**/
+
+#include <minIni.h>
+#include "ZuluSCSI_disk.h"
+#include "ZuluSCSI_log.h"
+#include "QuirksCheck.h"
+#include <stdint.h>
+
+
+static bool isValidMacintoshImage(image_config_t *img)
+{
+    bool result = true;
+    const char apple_magic[2] = {0x45, 0x52};
+    const char block_size[2]  = {0x02, 0x00};  // 512 BE == 2
+    const char lido_sig[4] = {'C', 'M', 'S', '_' };
+    uint8_t tmp[4] = {0};
+
+    // Check for Apple Magic
+    img->file.seek(0);
+    img->file.read(tmp, 2);
```

Comment:
I think the reason why raw mappings are excluded in the `macQuirksSanityCheck()` is these unaligned accesses, which won't work because raw access must be in whole SD blocks. This ends up excluding any continuous image file.

This could be modified to be `uint8_t tmp[SD_SECTOR_SIZE]; img->file.seek(0); img.file.read(tmp, SD_SECTOR_SIZE);` and then comparing against specific bytes in the tmp array.